### PR TITLE
Stdout to additional print datetime in human friendly format

### PR DESCRIPTION
**Pull request by cijothomas**
Continuing https://github.com/open-telemetry/opentelemetry-rust/pull/1192 to logs and traces as well.

### CODE REVIEW

**Review by lalitb — APPROVED** [review]

File: opentelemetry-stdout/src/common.rs [hunk 1]
```diff
@@ -239,6 +240,40 @@ impl From<opentelemetry_sdk::Scope> for Scope {
...
+                .format("%Y-%m-%d %H:%M:%S.%3f")
+                .to_string()
+                .as_ref(),
+        ),
+        _ => Err(serde::ser::Error::custom("Invalid Timestamp.")),
+    }
+}
+
+#[allow(dead_code)]
+pub(crate) fn as_opt_human_readable<S>(
```
Comment (TommyCpp) [inline opentelemetry-stdout/src/common.rs:264]: maybe add a comment here saying it's for serde?

**Review by TommyCpp — APPROVED** [review]
One nit but otherwise LGTM

**Comment by cijothomas** [comment]
> One nit but otherwise LGTM

Done! 

There is a docs CI failure, unrelated though!

**Comment by cijothomas** [comment]
@TommyCpp ready to merge, CI is clean now!

**Comment by jtescher** [comment]
Looks good, just have to resolve the new conflicts

**Comment by cijothomas** [comment]
@jtescher fixed conflicts, ready to merge.

**Pull request merged**